src/main/java/io/smartcat/node/monkey/config/Configuration.java

```diff
@@ -25,4 +31,21 @@
     public static Configuration loadDefaults() {
         return new Configuration();
     }
+
+    /**
+     * Determines whether or not a configuration is valid.
+     *
+     * @return true if configuration is valid, false otherwise
+     */
+    boolean isValid() {
+        return (failedRequestsPercentage > 0 && delayedRequestsPercentage > 0)
```

Comment:
You should be able to have 0 failed or 0 delayed. Doesn't make sense to force them. Greater or equal makes sense.